test/parallel/test-global-descriptors.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+
+const assert = require('assert');
+
+const {
+  value,
+  configurable,
+  enumerable,
+  writable
+} = Object.getOwnPropertyDescriptor(global, 'global');
+
+assert.strictEqual(value, global, 'global should be global object');
```

Comment:
I think `Function('return this')()` is fine. (Technically, `function () {}.constructor('return this')()` could also cause a problem if `Function.prototype.constructor` was overridden, but I don't think we need to worry about that here.)